jwst/guider_cds/guider_cds.py

```diff
@@ -98,6 +129,72 @@ def guider_cds(model):
     return new_model
 
 
+def get_ref_arr(model, imshape, gain_model=None, readnoise_model=None):
+    """
+    Short Summary
+    -------------
+    Create gain and readnoise files from their reference files
+
+    Parameters
+    ----------
+    model: data model
+        input data model, assumed to be of type GuiderRawModel
```

Comment:
Again, specify the type of model on the first line. If it's assumed to be a `GuiderRawodel` just say so directly.

Reply:
So I guess line 139 should say "model : `datamodels.GuiderRawModel`" instead of "data model" and then no need to mention the model type in line 140.